File: unsu.go
package main

import (
	"fmt"
	"os"
	"os/exec"
	"regexp"
	"strings"
)

func run(command string) {
	cmd := exec.Command("bash", "-c", command)
	stdout, _ := cmd.Output()
	fmt.Println(string(stdout))
	return
}

func main() {
	r, _ := regexp.Compile("[sudo]{4}")
	command := strings.Join(os.Args[1:], " ")

	if r.MatchString(command) {
		run(strings.Replace(command, "sudo ", "", -1))
	} else {
		run(command)
	}
}

File: unsu.py
#!/usr/bin/python3

import os
import re
import argparse

parser = argparse.ArgumentParser()
parser.add_argument("cmd", nargs='+')

args, unknown = parser.parse_known_args()
command = ' '.join(args.cmd) + ' ' + ' '.join(unknown)

rx = re.compile("[sudo]{4}")
if rx.match(command):
    os.system(command.replace("sudo ", ""))
else:
    os.system(command)
